lib/features/sks-menu/data/repository/sks_menu_repository.dart

```diff
@@ -6,24 +6,21 @@ import "package:riverpod_annotation/riverpod_annotation.dart";
 import "../../../../shared_api_clients/sks_api_client.dart";
 import "../models/dish_category_enum.dart";
 import "../models/sks_menu_data.dart";
+import "../models/sks_menu_response.dart";
 
 part "sks_menu_repository.g.dart";
 
 @riverpod
 Future<IMap<DishCategory, List<SksMenuDish>>> getSksMenuData(Ref ref) async {
+  const mealsEndpoint = "/api/v1/meals/current";
```

Comment:
In my `.env` file SKS_URL is as follows: `SKS_URL="https://sks-api.topwr.solvro.pl/api/v1"` in other parts of code  endpoints are used that adapt to this format. 
So endpoint here should start with `/meals..`

Reply:
Okay